### PR TITLE
Added file listing down the bechmarks for reads

### PR DESCRIPTION
**Pull request by vadlakondaswetha**

### CODE REVIEW

File: docs/benchmarks.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# GCSFuse Performance Benchmarks
+
+FIO is used to perform load tests on GCSFuse. Below tables shows performance metrics of GCSFuse for different workloads for the given test setup:
```
Comment (avidullu) [inline docs/benchmarks.md:3]: Can link to https://fio.readthedocs.io/en/latest/ ?

File: docs/benchmarks.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+### FIO Spec
+* Test runtime: 60sec
+* Thread count: 40
+* Block Size: 1MB
+
+### GCSFuse command
+```
+gcsfuse  --max-conns-per-host=100 --implicit-dirs --stat-cache-ttl=60s 
+--type-cache-ttl=60s --disable-http2 <bucket-name> ~/gcs
```
Comment (avidullu) [inline docs/benchmarks.md:23]: Maybe say LOCAL_MOUNT_POINT?

File: docs/benchmarks.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+```
+## Reads
+### Sequential reads
```
Comment (avidullu) [inline docs/benchmarks.md:26]: Can we give the command as well similar to https://cloud.google.com/compute/docs/disks/benchmarking-pd-performance#existing-disk ?

File: docs/benchmarks.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+| 10MB      | 1.44                 | 1435  | 1761               |
+| 50MB      | 1.31                 | 1301  | 1934               |
+| 100MB     | 1.07                 | 1056  | 2365               |
+| 200MB     | 1.06                 | 1044  | 2405               |
+| 1GB       | 0.91                 | 889   | 2809               |
+
+
+### Recommendation for reads
+GCSFuse performs well for sequential reads and recommendation is to use GCSFuse
+for doing sequential reads on file sizes > 10MB and < 1GB.
```
Comment (avidullu) [inline docs/benchmarks.md:57]: Can we also include the flags (max-conns-per-host , --disable-http2 etc.) which can be used to further improve gcsfuse performance?

**Review by avidullu — APPROVED** [review]

File: docs/benchmarks.md (hunk 3 above)
Comment (vadlakondaswetha) [reply docs/benchmarks.md:26]: done

File: docs/benchmarks.md (hunk 1 above)
Comment (vadlakondaswetha) [reply docs/benchmarks.md]: done

File: docs/benchmarks.md (hunk 4 above)
Comment (vadlakondaswetha) [reply docs/benchmarks.md]: done

**Pull request merged**